src/views/use/routeWatcher.ts

```diff
@@ -84,7 +84,15 @@ const useRouteWacher = (context: SetupContext) => {
       state.view = 'not-found'
       return
     }
-    changeViewTitle(`#${state.channelParam}`)
+    const id = channelPathToId(
+      state.channelParam.split('/'),
+      store.state.domain.channelTree.channelTree
+    )
+    const useShortenedPath = (props: any) => {
+      const { channelIdToShortPathString } = useChannelPath()
+      return channelIdToShortPathString(id)
+    }
+    changeViewTitle(`#${useShortenedPath(state.channelParam)}`)
```

Comment:
```suggestion
    const { channelIdToShortPathString } = useChannelPath()
    changeViewTitle(`#${channelIdToShortPathString(id)}`)
```
これはこういう風に書き換えられますね

---
元のコード
```ts
    const useShortenedPath = (props: any) => {
      const { channelIdToShortPathString } = useChannelPath()
      return channelIdToShortPathString(id)
    }
    changeViewTitle(`#${useShortenedPath(state.channelParam)}`)
```
propsは使ってないので受け取らなくてもよいので消す
```ts
    const useShortenedPath = () => {
      const { channelIdToShortPathString } = useChannelPath()
      return channelIdToShortPathString(id)
    }
    changeViewTitle(`#${useShortenedPath()}`)
```
この行は外に出してもよいので一旦出す
```ts
    const { channelIdToShortPathString } = useChannelPath()
    const useShortenedPath = () => {
      return channelIdToShortPathString(id)
    }
    changeViewTitle(`#${useShortenedPath(state.channelParam)}`)
```
`useShortenedPath`はただ`channelIdToShortPathString(id)`をしてるだけなのでそのまま使うようにする
```ts
    const { channelIdToShortPathString } = useChannelPath()
    changeViewTitle(`#${channelIdToShortPathString(id)}`)
```